src/styles/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$main-accent: #060b35;
+$secondary: #616070;
+$white: #fff;
+$blue-accent: #00acdc;
+$gray: #e5e5e5;
+$container-padding-x: 15px;
```

Comment:
```suggestion
```

Why you need this var? You dont using gray, and container-padding-x using only one time.